mkosi/__init__.py

```diff
@@ -4735,8 +4744,12 @@ def __init__(self, *kargs: Any, **kwargs: Any) -> None:
         self._ini_file_list_mode = False
 
         # Add config files to be parsed
-        kwargs["fromfile_prefix_chars"] = ArgumentParserMkosi.fromfile_prefix_chars
-        super().__init__(*kargs, **kwargs)
+        super().__init__(
+            *kargs,
+            **kwargs,
+            fromfile_prefix_chars=ArgumentParserMkosi.fromfile_prefix_chars,
+            formatter_class=CustomHelpFormatter,
+        )
```

Comment:
mypy errors here because it can't generally prove, that kwargs will never have those keys, which would lead to an error at runtime. This doesn't have the same "problem" (not that practically would be an issue) and maybe conveys the the intention a bit better.
```suggestion
        kwargs["fromfile_prefix_chars"] = ArgumentParserMkosi.fromfile_prefix_chars
        kwargs["formatter_class"] = CustomHelpFormatter

        super().__init__(*kargs, **kwargs)
```